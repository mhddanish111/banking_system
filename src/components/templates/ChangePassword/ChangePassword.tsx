import React from 'react';
import Layout from '../Layout';
import ChangePasswordForm from '../../molecules/ChangePasswordForm';

const ChangePassword: React.FC<any> = () => {
  return (
    <Layout pageHeading="Change Password">
      <ChangePasswordForm />
    </Layout>
  );
};

export default ChangePassword;
