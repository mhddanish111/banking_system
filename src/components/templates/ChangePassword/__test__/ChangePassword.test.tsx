import { shallow } from 'enzyme';
import ChangePassword from '../ChangePassword';

describe('<ChangePassword />', () => {
  const wrapped = shallow(<ChangePassword />);
  it('should render the ChangePassword Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
