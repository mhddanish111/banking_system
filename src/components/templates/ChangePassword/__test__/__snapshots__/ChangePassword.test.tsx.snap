// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangePassword /> should render the ChangePassword Component correctly 1`] = `
<Layout
  pageHeading="Change Password"
>
  <ChangePasswordForm />
</Layout>
`;
